manager-bundle/src/Command/ContaoSetupCommand.php

```diff
@@ -89,7 +102,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             $this->executeCommand(array_merge($php, [$this->consolePath], $command, $commandFlags), $output);
         }
 
-        $output->writeln('<info>Done! Please run the contao:migrate command to make sure the database is up-to-date.</info>');
+        $io->success('Done! Please run the contao:migrate command to make sure the database is up-to-date.');
```

Comment:
I guess that should be `$io->info`

Reply:
See 5c3802e840a2c229a81cf54202ef6a56b474ef8f.